app/views/admin/overview/_analytics.html.erb

```diff
@@ -61,7 +61,7 @@
           data-values="<%= values.join(",") %>"></canvas>
         <footer class="flex justify-between items-center fs-xs color-secondary">
           <span><%= @labels[0] %></span>
-          <span>Today</span>
+          <span>Yesterday</span>
```

Comment:
Do we want to localize this?

Reply:
Great question! 

We're building out the solution in the public facing site first - while this is a good target for localization in the future, right now /admin/ pages aren't being localized.